src/connectors/hits-per-page/connectHitsPerPage.js

```diff
@@ -115,7 +114,7 @@ export default function connectHitsPerPage(renderFn, unmountFn) {
       );
     }
 
-    const defaultValue = find(userItems, item => item.default === true);
+    const defaultValue = userItems.find(item => item.default === true);
```

Comment:
I think userItems can be undefined

Reply:
We throw an error above if `items` (which is `userItems`) is not an array. It should be safe to use `find`.

https://github.com/algolia/instantsearch.js/blob/9d33599d0d95946bd8fba871b7ef0477e57c3614/src/connectors/hits-per-page/connectHitsPerPage.js#L104-L108

Reply:
It cannot because of those lines:

https://github.com/algolia/instantsearch.js/blob/9d33599d0d95946bd8fba871b7ef0477e57c3614/src/connectors/hits-per-page/connectHitsPerPage.js#L102-L108

The flow is hard to follow though.